kubetest/kind/kind.go

```diff
@@ -350,9 +345,9 @@ func (d *Deployer) TestSetup() error {
 // Down tears down the cluster.
 func (d *Deployer) Down() error {
 	cmd := exec.Command("kind", "delete", "cluster")
-	if err := d.control.FinishRunning(cmd); err != nil {
-		return err
-	}
+	// This function seems to be called even for --up only,
```

Comment:
@krzyzacy ??

Reply:
For the case here, check `IsUp()` first?

Reply:
clarification from sen: `--up` will try to tear down the previous cluster, so we should make sure IsUp is respected

Reply:
we can revisit this later